biome.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "files": {
+    "ignore": [
+      "CHANGELOG.md",
+      "pnpm-lock.yaml",
+      "tsconfig.json",
+      "tsconfig.*.json",
+      "examples/*",
+      "*-app"
+    ]
+  },
+  "formatter": {
+    "enabled": true,
+    "formatWithErrors": false,
+    "indentStyle": "space",
+    "indentSize": 2,
+    "lineWidth": 80
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "recommended": true,
+      "correctness": {
+        "noUnusedVariables": "error"
+      },
+      "nursery": {
+        "noBannedTypes": "off",
+        "useExhaustiveDependencies": "error",
+        "useGroupedTypeImport": "off"
+      },
+      "performance": {
+        "noDelete": "off"
+      },
+      "style": {
+        "noNonNullAssertion": "off",
+        "useShorthandArrayType": "error",
+        "useSingleVarDeclarator": "off"
+      },
```

Comment:
Looks really good 🙏. I don't have any feedbacks on the `biome.json` file apart form maybe having : `"noDuplicateParameters": "off"` ? [Link](https://biomejs.dev/linter/rules/no-duplicate-parameters)